contentcuration/contentcuration/tests/test_garbage_collect.py

```diff
@@ -141,16 +143,18 @@ def test_doesnt_delete_file_referenced_by_orphan_and_nonorphan_nodes(self):
 
         # our legit node, standing proud and high with its non-orphaned status
         legit_node = ContentNode.objects.create(
-            kind_id="Video",
+            kind_id=content_kinds.VIDEO,
         )
 
         f = File.objects.create(
             contentnode=legit_node,
             checksum="aaa",
+            file_format_id=file_formats.JPG
```

Comment:
needed for saving the file, otherwise something tries to guess that `file_format_id='.jpg'` when it should be 'jpg'